packages/components/src/components/Alert/Alert.tsx

```diff
@@ -29,15 +23,15 @@ export type AlertProps = {
   /** Optional custom content to nest inside Alert
    * Use AlertContentColor or appropriate component props to set light/dark mode 'base' gray colors */
   children?: React.ReactNode
+  /** Optional boolean to determine whether the alert should be expandable */
```

Comment:
Given it's a boolean, may be clearer doc to just jump straight to something like: 
`True to make the Alert expandable/collapsible`